mbed_greentea/mbed_test_api.py

```diff
@@ -131,7 +131,7 @@ def run_host_test(image_path,
     @param duration Test case timeout
     @param micro Mbed-enabled device name
     @param reset Reset type
-    @param reset_tout Reset timeout (sec)
+    @param forced_reset_timeout Reset timeout (sec)
```

Comment:
Can you also remove the `reset_tout` parameter from the function definition?

Reply:
done